Library/OptimizelyClientTypeTests.swift

```diff
@@ -76,4 +76,100 @@ final class OptimizelyClientTypeTests: TestCase {
       XCTAssertTrue(mockClient.getVariantPathCalled)
     }
   }
+
+  func testGetVariation() {
+    let mockOptimizelyClient = MockOptimizelyClient()
+      |> \.experiments .~
+      [OptimizelyExperiment.Key.pledgeCTACopy.rawValue: OptimizelyExperiment.Variant.variant1.rawValue]
+
+    withEnvironment(currentUser: User.template, optimizelyClient: mockOptimizelyClient) {
+      let variation = mockOptimizelyClient.getVariation(for: .pledgeCTACopy)
```

Comment:
I used a fake experiment here because I didn't want to have to update the tests when we remove the experiment

Reply:
Good idea! Will update.